app/models/piece.rb

```diff
@@ -2,6 +2,11 @@ class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
 
+  def move_piece(new_x, new_y)
```

Comment:
@ilyakrasnov That makes total sense, but there is a move_to! method in `piece.rb` already in the master, created as part of the capture logic (line 113):
```
def move_to!(new_x, new_y)
    current_game = self.game_id
    piece_to_capture = Piece.find_by(x_cord: new_x, y_cord: new_y, game_id: current_game)

    return if piece_to_capture && own_piece?(piece_to_capture)

    # if the rest of this method is executed, either
    #   * piece_to_capture is nil
    #   * piece_to_capture is opposite color

    piece_to_capture.update(x_cord: nil, y_cord: nil) if piece_to_capture
    self.update(x_cord: new_x, y_cord: new_y)
  end
```
Do @ruishine and I need to reconcile these two into one method at this point?